rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParametricTextDocumentService.java

```diff
@@ -113,6 +114,7 @@
 import io.usethesource.vallang.IValue;
 import io.usethesource.vallang.IWithKeywordParameters;
 
+@SuppressWarnings({"deprecation"})
```

Comment:
can we add a comment that says why we are adding a deprecation warning? or a TODO on what we need to?